v-next/hardhat/package.json

```diff
@@ -60,6 +61,7 @@
     "src/",
     "templates/",
     "console.sol",
+    "coverage.sol",
```

Comment:
I don't think this is needed.

Reply:
You're right! I added it here before doing the trick with modifying solc inputs.